walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -69,12 +69,13 @@
       return slotsForMandatory;
     }
 
-    /* compute the intersection between the two sets of slots */
-    Collection<TimeRange> intersection = intersection(slotsForOptional, slotsForMandatory);
-    if (intersection.isEmpty()) {
+    /* compute the slots that accommodate both optional and mandatory attendees */
+    Collection<TimeRange> overallAvailableSlots =
+        overallAvailableSlots(slotsForOptional, slotsForMandatory, request.getDuration());
```

Comment:
Careful, the function signature is `overallAvailableSlots(Collection<TimeRange> mandatory, Collection<TimeRange> optional, long duration)` but here you call it with `slotsForOptional` first.